airflow/models/dagrun.py

```diff
@@ -140,6 +140,7 @@ class DagRun(Base, LoggingMixin):
         default=select(func.max(LogTemplate.__table__.c.id)),
     )
     updated_at = Column(UtcDateTime, default=timezone.utcnow, onupdate=timezone.utcnow)
+    clear_number = Column(Integer, default=0)
```

Comment:
```suggestion
    clear_number = Column(Integer, default=0, nullable=False)
```

to match the migration?